.github/workflows/update-python-dependencies.yml

```diff
@@ -0,0 +1,26 @@
+name: Update Python dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  "12 6 * * *"
```

Comment:
AFAICT dependabot runs weekly, yet this runs at 0612h every day - is there a reason for this change?

Reply:
I think this was taken from an existing repository, so I can change it to weekly.

Reply:
Done.